File: leetcode-submissions/2220_Minimum_Bit_Flips_to_Convert_Number.ts
function minBitFlips(start: number, goal: number): number {
    let xor_value= start ^ goal;
    let ans:number=0;
    while(xor_value>0){
        ans+=xor_value & 1;
        xor_value>>=1;
    }
    return ans;

};

File: leetcode-submissions/2220_Minimum_Bit_Flips_to_Convert_Number.cpp
class Solution {
public:
    int minBitFlips(int start, int goal) {
        int xor_value=start ^ goal;
        int ans=0;
        while(xor_value>0){
            ans+=xor_value & 1 ;
            xor_value=xor_value>>1;
        }
        return ans;
    }
};